allennlp/models/model.py

```diff
@@ -287,13 +287,22 @@ def _load(
         )
 
         model_params = config.get("model")
+        extras = {}
+        if "transformers_from_pretrained_kwargs" in config:
+            extras["transformers_from_pretrained_kwargs"] = config[
+                "transformers_from_pretrained_kwargs"
+            ].as_dict()
 
         # The experiment config tells us how to _train_ a model, including where to get pre-trained
         # embeddings from.  We're now _loading_ the model, so those embeddings will already be
         # stored in our weights.  We don't need any pretrained weight file anymore, and we don't
         # want the code to look for it, so we remove it from the parameters here.
         remove_pretrained_embedding_params(model_params)
-        model = Model.from_params(vocab=vocab, params=model_params)
+        model = Model.from_params(
+            vocab=vocab,
+            params=model_params,
+            **extras,
+        )
```

Comment:
Shouldn't need this either

```suggestion
        model = Model.from_params(vocab=vocab, params=model_params)
```